UseCases/Sources/InstrumentsAppTracingInstrpkg/SpansExampleInstrument/SpansExampleInstrument.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,166 @@
+// !$*UTF8*$!
```

Comment:
I guess we need the Xcode project files for Signpost, right?

Reply:
For the signpost APIs no, but for the `instrpkg` definition yes.